### PR TITLE
Oralndo - a dream place to visit

### PR DESCRIPTION
**Pull request by ROlety**
Visit Orlando to enjoy Disney parks
HI @githubteacher , I am orlando too.

### CODE REVIEW

File: ro-orlando [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ testing:wq
+### Restaurants
+- Woodlands
+
+### attractions
+Lego Land
```
Comment (MarkyStuart) [inline ro-orlando]: This must be so awesome. @ROlety

File: ro-orlando [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## Disney parks
+Epcot
```
Comment (MarkyStuart) [inline ro-orlando]: What is this?

File: ro-orlando [hunk 3]
```diff
@@ -0,0 +1,9 @@
+ testing:wq
+### Restaurants
+- Woodlands
+
+### attractions
+Lego Land
+
+## Disney parks
+Epcot - Disney park with world showcase and all display of advanced sceince technolgy.
```
Comment (MarkyStuart) [inline ro-orlando]: Thanks!!!!

**Comment by ROlety** [comment]
Merge test

**Pull request merged**